Website/DesktopModules/Hotcakes/Core/HotcakesController.cs

```diff
@@ -152,6 +152,10 @@ public string UpgradeModule(string Version)
                     case "03.05.00":
                     case "03.06.00":
                         UpdateStoreSettings();
+                        break; 
+                    
+                    case "03.08.02":
```

Comment:
@alejoroman0605 Please update the version to `03.09.00`.

Reply:
@WillStrohl I have already updated the version and also created a new provider for version 03.09.00.SqlDataProvider, because I realized that the creation of the "Daily Online Order Payment Failure Report" template was in a provider of a previous version.